.github/workflows/chromatic.yml

```diff
@@ -1,21 +1,35 @@
-name: "Chromatic"
+name: Chromatic
 on: push
 jobs:
   publish:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      # https://github.com/actions/checkout
+      - uses: actions/checkout@v4
         with:
           fetch-depth: 0
-      - name: Install Node.js
-        uses: actions/setup-node@v3
+
+      # https://nodejs.org/api/corepack.html
+      - name: Enable corepack (for pnpm)
+        run: corepack enable
+
+      # https://github.com/actions/setup-node
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
         with:
+          cache: pnpm
           node-version: 20
-      - name: Install dependencies
-        uses: pnpm/action-setup@v2
-        with:
-          version: 8
-          run_install: true
+
+      # https://pnpm.io/cli/install
+      - name: Install Node.js dependencies with pnpm
+        run: >
+          pnpm install
+          --frozen-lockfile
+
+      - name: Build package
+        run: >
+          pnpm run build
```

Comment:
Why is this necessary? I think that the addon is only using relative references in this repo, so building Storybook (which Chromatic always does) should just build the addon.

I'm fine with this step here, I'm just wondering if needing it is a symptom of something else I'm missing.

Reply:
It was failing, before I added this step. There's the log from the CI workflow: 
https://github.com/storybookjs/addon-svelte-csf/actions/runs/9073717361/job/24931260330#step:6:97

Reply:
Ah yes, probably because of this:

> WARN Could not resolve addon "../dist/preset/index.js", skipping. Is it installed?

I have no idea why this worked before..